doc/getting-started.rst

```diff
@@ -0,0 +1,593 @@
+
+  to start a server and::
+
+    bin/server stop
+
+  to stop it.  The script references a zdaemon configuration file
+  generated by the recipe in ``parts/server/zdaemon.conf``.
+
+- A zdaemon configuration script is generated in
+  ``parts/server/zdaemon.conf`` that looks something like::
+
+    <runner>
+      daemon on
+      directory /Users/jim/t/0214/parts/server
+      program /Users/jim/t/0214/bin/runzeo -f /Users/jim/t/0214/data.fs -a 127.0.0.1:8200
+      socket-name /Users/jim/t/0214/parts/server/zdaemon.sock
+      transcript /Users/jim/t/0214/parts/server/transcript.log
+    </runner>
+
+    <eventlog>
+      <logfile>
+        path /Users/jim/t/0214/parts/server/transcript.log
+      </logfile>
+    </eventlog>
+
+  The **details aren't important**, other than the fact that the
+  configuration file reflects part options and the actual buildout
+  location.
+
+Version control
+===============
+
+In this example, the only file that needs to be checked into version
+control is the configuration file, ``buildout.cfg``.  Everything else
+is generated.  Someone else could check out the project, and get the
+same result [#if-same-environment]_.
+
+More than just a package installer
+==================================
+
+The example shown above illustrates how Buildout is more than just a
+package installer, like ``pip``. Using Buildout recipes, we can
+install custom scripts and configuration files, and much more. For
+example, we could use `configure and make
+<https://pypi.python.org/pypi/zc.recipe.cmmi>`_ to install non-Python
+software from source, we could run JavaScript builders, or do anything
+else that can be automated with Python.
+
+Buildout is a simple automation framework.  There are hundreds of
+recipes to choose from and :doc:`writing new recipes is easy
+<topics/writing-recipes>`.
+
+Repeatability
+=============
+
+A major goal of Buildout is to provide repeatability.  But what does
+this mean exactly?
+
+  If two buildouts with the same configuration are built in the same
+  environments at the same time, they should produce the same result,
+  regardless of their build history.
+
+That definition is rather dense. Let's look at the pieces:
+
+Buildout environment
+--------------------
+
+A Buildout environment includes the operating system and the Python
+installation it's run with. The more a buildout depends on its
+environment, the more variation is likely between builds.
+
+If a Python installation is shared, packages installed by one
+application affect other applications, including buildouts. This can
+lead to unexpected errors.   This is why it's recommended to use a
+`virtual environment <https://virtualenv.pypa.io/en/stable/>`_ or a
+"clean python" built from source with no third-party packages
+installed.
+
+(It's a little hypocritical to recommend installing Buildout into an
+otherwise clean environment, which is why Buildout provides a
+:doc:`bootstrapping mechanism <topics/bootstrapping>` which allows
```

Comment:
Even if we deprecate it, I think we should talk about it given history.

We're definitely not going to emphasize it like we used to.

My plan is to mention that, because it depends on setuptools self-bootstrapping, it might go away in the future.

Buildout is opinionated and self-bootstrapping is an ideal consistent with Buildout's philosophy, even if it becomes impractical at some point.

As a compromise, I moved the mention of self-bootstrapping to a footnote.